### PR TITLE
Fix pointer on CommitBox not displaying

### PR DESCRIPTION
**Pull request by andrewhli**
PR request based on @fcollonval's suggestion from this thread: https://github.com/jupyterlab/jupyterlab-git/issues/1035#issuecomment-919755588

### CODE REVIEW

**Comment by andrewhli** [comment]
@fcollonval - Thanks for the tip. Could you please check the syntax? 
`Overload 1 of 2, '(...objects: NestedCSSProperties[]): string', gave the following error.
    Type '"auto !important"' is not assignable to type '"-moz-initial" | "inherit" | "initial" | "revert" | "unset" | "auto" | "none" | "visible" | "all" | "fill" | "stroke" | "painted" | "visibleFill" | "visiblePainted" | "visibleStroke" | PointerEvents[]'.`

File: src/style/CommitBox.ts [hunk 1]
```diff
@@ -109,5 +109,6 @@ export const activeStyle = style({
 export const disabledStyle = style({
   cursor: 'not-allowed !important',
   color: 'var(--jp-ui-font-color2) !important',
-  backgroundColor: 'var(--jp-layout-color3) !important'
+  backgroundColor: 'var(--jp-layout-color3) !important',
+  pointerEvents: 'auto !important'
```
Comment (fcollonval) [inline src/style/CommitBox.ts:113]: This is a limitation of the type. It seems it knows about `auto` but not with the flag `!important`. so let's trick ts on this one with a comment:

```suggestion
  // TypeScript does not know about the value with `!important` flag
  pointerEvents: 'auto !important' as any
```

**Review by fcollonval — APPROVED** [review]
Thanks @andrewhli

**Comment by fcollonval** [comment]
@meeseeksdev please backport to 0.32.x

**Pull request merged**